app/models/admin_set.rb

```diff
@@ -87,12 +87,12 @@ def active_workflow
   end
 
   ##
-  # @deprecated use PermissionTemplate#reset_access_controls instead
+  # @deprecated use PermissionTemplate#reset_access_controls_for instead
   #
   # Calculate and update who should have edit access based on who
   # has "manage" access in the PermissionTemplateAccess
   def reset_access_controls!
-    Deprecation.warn("reset_access_controls! is deprecated; use PermissionTemplate#reset_access_controls instead.")
+    Deprecation.warn("reset_access_controls! is deprecated; use PermissionTemplate#reset_access_controls_for instead.")
```

Comment:
There are a number of deprecated methods that were updated to use `reset_access_controls_for` but the deprecation message said to use `reset_access_controls`.  These were updated.